eksperymenty_programistyczne_na_edoku4.rst

```diff
@@ -0,0 +1,91 @@
+Eksperymenty programistyczne na eDoku4
+======================================
+
+Pierwszy eksperyment programistyczny
+------------------------------------
+
+Cel eksperymentu
+~~~~~~~~~~~~~~~~
+
+Celem eksperymentu jest wstępne wysondowanie możliwości rozbudowy testowanego systemu. Jego wynik może pomóc w opracowaniu kolejnych, bardziej wyszukanych eksperymentów.
+
+Założenia eksperymentu
+~~~~~~~~~~~~~~~~~~~~~~
+
+Instrukcja kancelaryjna w ramach minimalnego zestawu metadanych opisujących przesyłki wpływające oraz wychodzące wymienia (opcjonalny) adres email odpowiednio nadawcy oraz adresata. Urząd mógłby chcieć wdrożyć narzędzie umożliwiające Obywatelom zalogowanie się swoim adresem email oraz wyświetlenie przesyłek powiązanych ze swoim kontem. Eksperyment ma za zadanie ustalić możliwość łatwego napisania takiej aplikacji.
+
+Forma eksperymentu
+~~~~~~~~~~~~~~~~~~
+
+W ramach eksperymentu zbadana zostanie możliwość wydobycia z bazy danych poszczególnych systemów EZD metadanych przesyłek powiązanych z konkretnym adresem email. Pierwszoplanową metodą osiągnięcia tego celu będzie skorzystanie z API; w dalszej kolejności rozpatrywane będą inne metody, o ile nie będą się one wiązały z nadmierną uciążliwością programistyczną. Jeżeli uzyskanie metadanych przesyłek okaże się możliwe, zbadana zostanie także możliwość uzyskania odwzorowań cyfrowych tych przesyłek.
```

Comment:
Jeśli traktujemy to słowo jako przeniesione z angielskiego, to [Donald Knuth zaleca pisownię „email” bez łącznika](http://www-cs-faculty.stanford.edu/~uno/email.html). [Prof. Mirosław Bańko zachęca z kolei do pisowni „mejl”](http://sjp.pwn.pl/poradnia/haslo/adres-e-mail-czy-e-mailowy;11882.html), ale wydaje się ona obrzydliwa językowo. Tygodnik „Polityka” swego czasu promował „listel”. Osobiście polecałbym pisownię „list elektroniczny”, w skrócie „elist”. W tym wypadku chodziłoby o „adres poczty elektronicznej”, ew.  w skrócie „adres epoczty”.